client/app/bundles/course/assessment/submission/components/answers/Programming/index.jsx

```diff
@@ -122,29 +121,51 @@ const Programming = (props) => {
   );
 
   return (
-    <div className="mt-5">
-      {fileSubmission ? (
-        <ProgrammingImportEditor
-          key={question.id}
-          answerId={answerId}
-          isSavingAnswer={isSavingAnswer}
-          question={question}
-          readOnly={readOnly}
-          saveAnswerAndUpdateClientVersion={saveAnswerAndUpdateClientVersion}
-        />
-      ) : (
-        <ProgrammingFiles
-          key={question.id}
-          answerId={answerId}
-          feedbackFiles={feedbackFiles}
-          language={parseLanguages(question.language)}
-          questionId={question.id}
-          readOnly={readOnly}
-          saveAnswerAndUpdateClientVersion={saveAnswerAndUpdateClientVersion}
-        />
-      )}
+    <>
+      <div className="mt-5 flex w-full relative gap-3 mb-1 max-h-[100%]">
+        <Box
```

Comment:
nit: if `Box` is not needed then replace with a `div` whenever you come across them ([docs](https://github.com/Coursemology/coursemology2/blob/master/client/README.md#-do-not-use-muis-box-component)).

Reply:
done